content/blog/20250501_fluentd-v1.16.8-has-been-released.md

```diff
@@ -0,0 +1,43 @@
+# Fluentd v1.16.8 has been released
+
+Hi users!
+
+We have released v1.16.8 on 2025-05-01.
+ChangeLog is [here](https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1168---20250501).
+
+This release is a maintenance release of v1.16 series.
+
+This release will be bundled for upcoming `fluent-package` LTS version v5.0.7!
+
+## Bug Fixes
+
+### Windows: Stop the service correctly when the supervisor is dead
+
+In this release, we have fixed a bug that Windows service process will continue running
+even if the supervisor process is dead due to config error or etc.
+
+It is easier to notice config errors and other issues by this issue fixed.
+
+### `formatter_csv`: Fixed memory leak
+
+The caching mechanism introduced for performance optimization that was caching threads inappropriately in `formatter_csv`.
```

Comment:
> The caching mechanism introduced for performance optimization that was caching threads inappropriately in `formatter_csv`.

I find it difficult to read this sentence.
Which is the verb?

Reply:
Subject is `The caching mechanism introduced for performance optimization`.

Reply:
verb is `was caching`

Reply:
Removed